PlayerTwo/canvas.js

```diff
@@ -0,0 +1,52 @@
+// canvas
+const Y_AXIS_COMPENSATION = - 85
+const X_AXIS_COMPENSATION = - 4
+
+
+window.onload = () => {
+
+    const canvas = document.querySelector("#canvas");
+    const ctx = canvas.getContext("2d");
+    canvas.height = 400;
+    canvas.width = 500;
+    let brush = true;
+    let drawing = false;
+
+
+}
```

Comment:
Instead of using an event listener like before, this just renders the canvas on loading the page but doesn't call any functions